tests/Predis/ClientTest.php

```diff
@@ -1343,6 +1343,7 @@ public function testClientsCreateDifferentPersistentConnections(): void
     /**
      * @group connected
      * @group cluster
+     * @group relay-incompatible
```

Comment:
Relay sets `default` as username if only password is specified.

Reply:
Yeah, I think the problem is not in this test, Relay cluster setup just randomly fails probably cause infrastructure sometimes not yet ready when tests are running